ansible_collections/arista/avd/examples/single-dc-l3ls/ansible.cfg

```diff
@@ -2,3 +2,4 @@
 inventory=inventory.yml
 jinja2_extensions = jinja2.ext.loopcontrols,jinja2.ext.do,jinja2.ext.i18n
 duplicate_dict_key=error
+collections_paths = ~/ansible-dev/ansible-avd/:~/ansible-dev/ansible-cvp:~/.ansible/collections:/usr/share/ansible/collections
```

Comment:
this should not be there for regular examples where we expect people to install the collections properly